app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,18 @@
+class CommentsController < ApplicationController
+
+  def new
+    @picture = Picture.find(params[:picture_id])
+    @comment = Comment.new
+  end
+
+  def create
+    @picture = Picture.find(params[:picture_id])
+    @picture.comments.create(comment_params)
+    redirect_to pictures_path
+  end
+
+  def comment_params
+    params.require(:comment).permit(:comment)
```

Comment:
I would advise against making the attribute of a class have the same name as the class. So `comment.comment` is a bit weird.

`comment.description` or `comment.body` is good.